src/main/java/com/puppycrawl/tools/checkstyle/gui/Main.java

```diff
@@ -37,6 +37,13 @@
     private static JFrame frame;
 
     /**
+     * Hidden constructor of the current utility class.
+     */
+    private Main() {
+        throw new IllegalStateException("Utility class constructor mustn't be called.");
```

Comment:
I don't think that throwing an exception is necessary. In majority of other places I think we have just empty private constructor.